src/include/catalog/pg_appendonly.h

```diff
@@ -34,6 +34,7 @@ CATALOG(pg_appendonly,6105,AppendOnlyRelationId)
     bool            columnstore;        /* true if orientation is column */ 
     Oid             segrelid;           /* OID of aoseg table; 0 if none */
     int16           segfilecount;		/* the (per seg) average total number of segment file */
+	int16			version;			/* AO relation version */
```

Comment:
align code

Reply:
Thanks for pointing out，I'll fix in the next commit